src/one_dragon/base/cv_process/cv_service.py

```
@@ -36,6 +75,10 @@ def __init__(self, od_ctx: OneDragonContext):
         self.ocr = od_ctx.ocr
         self.template_loader = od_ctx.template_loader
 
+        # 插件流水线目录列表 [(插件名, 目录路径), ...]，随插件加载刷新
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)